test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -52,45 +52,138 @@ build_branch()
+create_configs()
+{
+    branch_name=$1
+
+    file_name=$(get_config_file_name $branch_name)
+
+    if [ -f $file_name ] ; then
+        return
+    fi
+
+    echo "##################################################" >> $file_name
+    echo "runs.type=row" >> $file_name              # Temporarily disable column store tests
+    echo "btree.prefix=0" >> $file_name             # Prefix testing isn't portable between releases
+    echo "cache=80" >> $file_name                   # Medium cache so there's eviction
+    echo "checksum=on" >> $file_name
+    echo "checkpoints=1"  >> $file_name             # Force periodic writes
+    echo "compression=snappy"  >> $file_name        # We only built with snappy, force the choice
+    echo "data_source=table" >> $file_name
+    echo "huffman_key=0" >> $file_name              # Not supoprted by newer releases
+    echo "in_memory=0" >> $file_name                # Interested in the on-disk format
+    echo "leak_memory=1" >> $file_name              # Faster runs
+    echo "logging=1" >> $file_name                  # Test log compatibility
+    echo "logging_compression=snappy" >> $file_name # We only built with snappy, force the choice
+    echo "rows=1000000" >> $file_name
+    echo "salvage=0" >> $file_name                  # Faster runs
+    echo "timer=4" >> $file_name
+    echo "verify=0" >> $file_name                   # Faster runs
+
+    # Append older release configs for newer compatibility release test
+    if [ $newer = true ]; then
+        for i in "${compatible_upgrade_downgrade_releases[@]}"
+        do
+            if [ "$i" == "$branch_name" ] ; then
+                echo "transaction.isolation=snapshot" >> $file_name # Older releases can't do lower isolation levels
+                echo "transaction.timestamps=1" >> $file_name       # Older releases can't do non-timestamp transactions
+                break
+            fi
+        done
+    fi
+    echo "##################################################" >> $file_name
+}
+
+#############################################################
+# create_configs_older_standalone_release:
+#############################################################
+create_configs_older_standalone_release()
+{
+    # Iterate over the releases and create configuration files
+    for b in *; do
+        if [ -d "$b" ]; then
+            (create_configs $b)
+        fi
+    done
+}
+
+#############################################################
+# create_configs_per_newer_release:
+#############################################################
+create_configs_per_newer_release()
+{
+    # Create configs for all the newer releases
+    for b in ${newer_release_branches[@]}; do
+        (create_configs $b)
+    done
+
+    # Copy per-release configs in the newer release branches
+    for b in ${newer_release_branches[@]}; do
+        cp -rf CONFIG* $b/test/format/
+    done
+
+    # Delete configs from the top folder
+    rm -rf CONFIG*
```

Comment:
Can this be combined with above line 150 with a `mv` command?

Reply:
No, It cannot be combined because the above `cp` command will copy the config files in every iteration to all the release branches.

Reply:
Thanks for clarification.